libs/api/domains/rsk-company-info/src/lib/graphql/rsk-company-info.service.ts

```diff
@@ -111,54 +121,21 @@ export class RskCompanyInfoService {
       } as RskCompany
     })
 
-    const pageResult = this.getPage(formattedSearchResults, first, after)
-    if (!pageResult) {
-      throw new NotFoundException('No search results found')
-    }
-
     const totalLength = searchResults.count
-    const cursor = toCursorHash(this.getLastItemIdentifer(pageResult.items))
+    const resultOffset = searchResults.offset ?? 0
+    const cursor = toBase64(
+      (resultOffset + formattedSearchResults.length).toString(),
+    )
 
     return {
-      items: pageResult.items,
-      count: totalLength,
+      data: formattedSearchResults,
+      totalCount: totalLength,
       pageInfo: {
         endCursor: cursor,
-        hasNextPage: pageResult.hasNextPage,
+        hasNextPage: !!searchResults.hasMore,
+        hasPreviousPage: resultOffset > 0,
+        startCursor: toBase64(resultOffset.toString()),
```

Comment:
@saevarma This API currently doesn't support paginating with "before". Did we decide that should be required for all APIs? :)

If not, we should get rid of "hasPreviousPage" and "startCursor" since they're not really useful if the API doesn't support "before".

Reply:
We did not make it optional fields in the documentation. But I think we should allow for the `hasPreviousPage` and `startCursor` to be optional for use cases like this. I will update the pagination docs.